doc/Extensions/RRDCached.md

```diff
@@ -225,13 +225,41 @@ ExecStart=/usr/bin/rrdcached -w 1800 -z 1800 -f 3600 -s librenms -U librenms -G
 WantedBy=default.target
 ```
 
-2: Start rrdcached
+2: Configure SELinux for RRDCached
+
+```
+echo 'module rrdcached_librenms 1.0;
```

Comment:
These multi line echo isn't really user friendly and can be broken go breaking when copy paste. Took a while to even see where it ended even. I'd suggest you use multi line string feature such as `<<EOF`